hugegraph-core/src/main/java/com/baidu/hugegraph/traversal/algorithm/HugeTraverser.java

```diff
@@ -60,13 +61,16 @@
 import com.baidu.hugegraph.util.CollectionUtil;
 import com.baidu.hugegraph.util.E;
 import com.baidu.hugegraph.util.InsertionOrderUtil;
+import com.baidu.hugegraph.util.Log;
 import com.baidu.hugegraph.util.collection.CollectionFactory;
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
 import com.google.common.collect.ImmutableSet;
 
 public class HugeTraverser {
 
+    public static final Logger LOG = Log.logger(HugeTraverser.class);
```

Comment:
used for all traverser subclass (dynamic debug or reused)

Reply:
marke as protected